acceptancetests/Makefile

```diff
@@ -1,31 +1,73 @@
-mkfile_dir := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
This Makefile change should be in a PR of it's own as it's not specific to the addition of the caas functional testing.

Reply:
yeah, can do. I will raise another PR for `Makefile` change later